src/network/netdev/bond.c

```diff
@@ -284,6 +285,24 @@ static int netdev_bond_fill_message_create(NetDev *netdev, Link *link, sd_netlin
                         return log_netdev_error_errno(netdev, r, "Could not append IFLA_BOND_MIN_LINKS attribute: %m");
         }
 
+        if (b->ad_actor_sys_prio != 0) {
+                r = sd_netlink_message_append_u16(m, IFLA_BOND_AD_ACTOR_SYS_PRIO, b->ad_actor_sys_prio);
+                if (r < 0)
+                        return log_netdev_error_errno(netdev, r, "Could not append IFLA_BOND_AD_ACTOR_SYS_PRIO attribute: %m");
+        }
+
+        if (b->ad_user_port_key != 0) {
+                r = sd_netlink_message_append_u16(m, IFLA_BOND_AD_USER_PORT_KEY, b->ad_user_port_key);
+                if (r < 0)
+                        return log_netdev_error_errno(netdev, r, "Could not append IFLA_BOND_AD_USER_PORT_KEY attribute: %m");
+        }
+
+        if (b->ad_actor_system != 0) {
```

Comment:
Please do not compare a pointer with '0'. `if (b->ad_actor_system) {` is ok here.

Reply:
@poettering hinted me once that the CODING_STYLE was to use explicit numeric comparisons. In case this was dropped shall I update the others as well in this file?

Reply:
No, `ad_actor_system` is a pointer, but the others you added are integer. Our coding style is, `if (pointer)` or `if (!pointer)` for pointer. And for integer e.g, `if (integer == 0)` or `if (integer != 0)`.

Reply:
Your completely right. Sorry for that.